recipes/default.rb

```diff
@@ -8,13 +8,27 @@
 include_recipe 'apt'
 include_recipe 'java'
 
-link "/etc/alternatives/java" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
-end
+#link "/etc/alternatives/java" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
+#end
+
+#link "/etc/alternatives/javac" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+#end
```

Comment:
I believe these symbolic links are required as well for the installation of Java 7 to be properly referenced by the system. Can you please uncomment them? Thanks!